src/components/new-request/search.vue

```diff
@@ -34,8 +34,33 @@
         <v-text-field filled disabled hide-details label="Select an action first" />
       </v-col>
 
+      <!-- Business Lookup/Fetch -->
+      <v-col v-if="showBusinessLookup" cols="12" md="6" class="business-lookup">
+        <template v-if="!business">
+          <BusinessLookup
+            v-if="getIsAuthenticated"
+            :businessStatus="fetchActiveOrHistorical"
```

Comment:
This is where whether we are checking if we'll be searching for Active or Historical. The function (line 410) checks if we're in the restoration action. If yes, we check for historical businesses.